lib/resources/auditd_rules.rb

```diff
@@ -15,6 +15,7 @@ def initialize(content)
         assignment_regex: /^\s*([^:]*?)\s*:\s*(.*?)\s*$/,
         multiple_values: true,
       }
+      @legacy = nil
```

Comment:
Good catch, I did put that in the wrong place. I moved that to the other initialize method.